test/TestProjects/FirstTest-Cadl/FirstTest-Cadl.cadl

```diff
@@ -63,7 +63,7 @@ model RoundTripModel {
 namespace Hello.Demo {
     @doc("Return hi")
     @get
-    op sayHi(): Thing;
+    op sayHi(@path pathParameter: "json" | "jpeg"): Thing;
```

Comment:
May I comprehend this type expression as a sealed choice that contains two values `json` and `jpeg`?